packages/cli/src/commands/info.js

```diff
@@ -0,0 +1,34 @@
+// inspired by gatsby/packages/gatsby-cli/src/create-cli.js and
+// and gridsome/packages/cli/lib/commands/info.js
+import envinfo from 'envinfo'
+
+export const command = 'info'
+export const desc = 'Prints your system environment information'
+export const builder = {
+  clipboard: {
+    alias: 'C',
+    type: 'boolean',
+    default: false,
+    describe: 'Copy info to clipboard',
+  },
+}
```

Comment:
```suggestion
```

Reply:
🤦‍♂ good catch… I’m moving to fast across many things today.Thanks!